setup.py

```diff
@@ -132,15 +132,16 @@ def run(self):
 
 
 setup_args['install_requires'] = [
-    'notebook>=4.3.1',
+    'notebook>=4.3.1,<5.7.5',
```

Comment:
I think this is okay for now, but we should not publish a release with it.

Reply:
Should we make an issue about this with the 1.0 milestone?

Reply:
Yes, absolutely.

Reply:
https://github.com/jupyterlab/jupyterlab/issues/6131